cli/cmd/plugin/unmanaged-cluster/cmd/utils.go

```diff
@@ -8,7 +8,10 @@ import (
 	"runtime"
 	"strconv"
 
+	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
+
```

Comment:
nit: Not sure why the lint check didn't complain, but there shouldn't be a separation here. `github.com/spf` and `golang.org/x` are both third party imports, so should be part of the same import grouping.